generated/nidigital/nidigital/session.py

```diff
@@ -2774,6 +2774,38 @@ def delete_all_time_sets(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def _file_helper(self, action, files):
+        if isinstance(files, str):
+            files = [files]
+        for f in files:
+            action(f)
+
+    def load_specifications_levels_and_timing(self, specifications_file_paths=None, levels_file_paths=None, timing_file_paths=None):
```

Comment:
specifications, levels are plural. Timing is singular. Shouldn't this be `timing_sheets_file_paths`?
If the function was called `load` would that be an improvement?
Interested in the corresponding names for VI and inputs.

Reply:
I followed the VI and input names used in [LabVIEW](https://www.ni.com/documentation/en/ni-digital/latest/ni-digital-node-ref/load-spec-levels-timing-multiple/); which unfortunately is not consistent in many ways.

We need to decide whether to be consistent with LabVIEW or break from it for the sake of better names.

I think `load` is too generic and isn't consistent with `load_patterns` and `load_pinmap`. In our documentation, `timing` is used with `levels` quite pervasively, we even have the [apply_levels_and_timing](https://nimi-python.readthedocs.io/en/master/nidigital/class.html#apply-levels-and-timing) public method.

We could update parameter names to `levels_sheets_file_paths` and `timing_sheets_file_path`; this would be a good improvement without being inconsistent. I like the extra context it provides about sheets and files that a new user would likely find confusing.

Reply:
ok